### PR TITLE
chore/upgrade manifest.js merging approach

### PR DESCRIPTION
**Pull request by PatrykKuniczak**
I want to propose a small upgrade, for simplify merging configs in manifest file :)

### CODE REVIEW

File: chrome-extension/manifest.js [hunk 1]
```diff
@@ -64,7 +65,7 @@ const manifest = Object.assign(
       },
     ],
   },
-  !isFirefox && { side_panel: { ...sidePanelConfig.side_panel } },
+  !isFirefox && sidePanelConfig,
```
Comment (Jonghakseo) [inline chrome-extension/manifest.js:68]:
```suggestion
  sidePanelConfig,
```
We already have logic for FireFox's sidePanel permission, so I think this is unnecessary.

**Review by Jonghakseo — COMMENTED** [review]
👍🏼

File: chrome-extension/manifest.js (hunk 1 above)
Comment (PatrykKuniczak) [reply chrome-extension/manifest.js:68]: OO, in this case, it's completely unnecessary, but i remember there's some warnings in console, without that, i need to check it 1 more time, and i will adjust code 😄

Comment (PatrykKuniczak) [reply chrome-extension/manifest.js:68]: image

It's still doesn't supported, this check need to be here

Comment (PatrykKuniczak) [reply chrome-extension/manifest.js:68]: I removed that unnecessary check in config

**Review by Jonghakseo — APPROVED** [review]
LGTM

**Pull request merged**